app/frontend/src/components/forms/manage/ManageVersions.vue

```diff
@@ -103,54 +159,82 @@
       @close-dialog="cancelPublish"
     >
       <template #title>
-        <span v-if="publishOpts.publishing">
-          Publish Version {{ publishOpts.version }}
-        </span>
+        <span v-if="publishOpts.publishing">Publish Version {{ publishOpts.version }}</span>
         <span v-else>Unpublish Version {{ publishOpts.version }}</span>
       </template>
       <template #text>
-        <span v-if="publishOpts.publishing">
-          This will set Version {{ publishOpts.version }} to the live form.
-        </span>
+        <span
+          v-if="publishOpts.publishing"
+        >This will set Version {{ publishOpts.version }} to the live form.</span>
         <span v-else>
           Unpublishing this form will take the form out of circulation until a
           version is published again.
         </span>
       </template>
     </BaseDialog>
+
+    <BaseDialog
+      v-model="showDeleteDraftDialog"
+      type="CONTINUE"
+      @close-dialog="showDeleteDraftDialog = false"
+      @continue-dialog="deleteCurrentDraft"
+    >
+      <template #title>Confirm Deletion</template>
+      <template #text>Are you sure you wish to delete this Version?</template>
+      <template #button-text-continue>
+        <span>Delete</span>
+      </template>
+    </BaseDialog>
   </div>
 </template>
 
 <script>
 import { mapActions, mapGetters } from 'vuex';
 
+import { formService } from '@/services';
 import { FormPermissions } from '@/utils/constants';
-import CurrentDraft from '@/components/forms/manage/CurrentDraft.vue';
 
 export default {
   name: 'ManageVersions',
-  components: { CurrentDraft },
   data() {
     return {
       headers: [
         { text: 'Version', align: 'start', value: 'version' },
         { text: 'Status', align: 'start', value: 'status' },
         { text: 'Last Update', align: 'start', value: 'updatedAt' },
         {
-          text: 'Create a New Version',
+          text: 'Actions',
           align: 'center',
-          value: 'create',
+          value: 'edit',
+          filterable: false,
+          sortable: false,
+        },
+        {
+          text: '',
+          value: 'export',
+          filterable: false,
+          sortable: false,
+        },
+        {
+          text: '',
+          value: 'action',
           filterable: false,
           sortable: false,
         },
       ],
+      formSchema: {
+        display: 'form',
+        type: 'form',
+        components: [],
+      },
       publishOpts: {
         publishing: true,
         version: '',
-        versionId: '',
+        id: '',
```

Comment:
i've relabeled this property so it makes sense as an identifier for both the 'version ID' and the 'draft ID'